src/app/core/components/left-header/left-header.component.html

```diff
@@ -1,16 +1,16 @@
 <div class="nav-header" *ngIf="!compactMode">
   <div class="nav-control">
-    <span class="nav-collapse" (click)="setFullFrameContent()" tabindex="0">
-      <i class="fas fa-expand-arrows-alt" style="font-size: 2rem;"></i>
-    </span>
     <span class="platform-info">
       <span class="platform-name" i18n>Algorea Platform</span>
     </span>
+    <span class="nav-collapse" (click)="setFullFrameContent()" tabindex="0">
+      <i class="fas fa-angle-double-left"></i>
+    </span>
   </div>
 </div>
 
 <div class="nav-control alt" *ngIf="compactMode">
   <span class="nav-collapse" (click)="unsetFullFrameContent()" tabindex="0">
-    <i class="fas fa-compress-arrows-alt" style="font-size: 2rem;"></i>
+    <i class="fas fa-angle-double-right"></i>
```

Comment:
When testing it, I think I prefer `fa-chevron-*` or `fa-chevron-circle-*` as the chevron is more commonly used for hiding/showing menu than the double chevron.
![Screenshot 2021-07-20 at 09 50 13](https://user-images.githubusercontent.com/1053150/126282720-9e7fd4b3-f5df-4c9b-bf1c-790ff41c691f.png)
![Screenshot 2021-07-20 at 09 49 58](https://user-images.githubusercontent.com/1053150/126282765-278f2454-9411-4ace-9ba3-a30ab6097fbb.png)

Reply:
Changed to `fa-chevron-circle`.